packages/swapper/src/swappers/ZrxSwapper/utils/fetchFromZrx.ts

```diff
@@ -65,7 +65,8 @@ export const fetchFromZrx = async <T extends 'price' | 'quote'>({
       sellAmount: sellAmountIncludingProtocolFeesCryptoBaseUnit,
       takerAddress: receiveAddress,
       affiliateAddress: AFFILIATE_ADDRESS, // Used for 0x analytics
-      skipValidation: priceOrQuote === 'price', // don't validate allowances for price queries
+      // Always skip validation, so that we can get a quote even if no wallet is connected
+      skipValidation: true,
```

Comment:
if we skip this on quote, do we still get back `estimatedGas`? I would think validation is the simulation of the tx that would also provide that, but I could be mistaken. We do use and need `estimatedGas` back from the quote as we were getting failed trades without it.

Reply:
Regular ZRX quotes have been disabled on all envs as of yesterday (permit2 enabled),and we need a bit of different parsing there which has been done in https://github.com/shapeshift/web/pull/7939/commits/db607784b4d037d022afaf39ab71e79fa571c104 but yes, price response for regular ZRX quotes include `estimatedGas` with skipped validation! https://paw.pt/jabNXKeV

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/9f568a4c-2b5c-46c2-a84c-25d0a8a79005">

And with permit2 enabled: https://paw.pt/jabP3abI

<img width="1725" alt="image" src="https://github.com/user-attachments/assets/56675528-bc3a-4b50-8c84-fbc09e7281ab">

Reply:
Just making a note that the price `estimatedGas` is most likely a rough estimation versus an actual estimation with `validation`. I remember seeing a diff between the two states on either 0x or Lifi (can't exactly recall which). Just want to validate this change since it is changing behavior from prod and we can still skip validation on price and use it on quote as it was.